drivers/espi/espi_mchp_xec.c

```diff
@@ -758,14 +870,30 @@ static void espi_oob_up_isr(struct device *dev)
 static void espi_flash_isr(struct device *dev)
 {
 	u32_t status;
+	struct espi_xec_data *data = (struct espi_xec_data *)(dev->driver_data);
+	struct espi_event evt = { ESPI_BUS_EVENT_CHANNEL_READY, 0, 0 };
```

Comment:
We should be using C99 initializaton in for structs as we have been doing that in the other drivers.